src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
@mullr I think "maybe" it could be useful if you had a wrapper that also did something with the error depending on the namespace, but I haven't seen that in any of the projects, so....